scripts/tempSensor/main.py

```diff
@@ -0,0 +1,83 @@
+import json
+import time
+
+from bme680 import BME680_I2C
+from machine import I2C, Pin, reset
+from netman import connectWiFi
+from umqtt.simple import MQTTClient
+
+# Configuration
+SSID = "Pixel 8"
```

Comment:
Please put all of this in a .gitgnore'd my_secrets.py file. You're exposing credentials publicly by committing.